itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/yahooweather.bundle/OH-INF/i18n/yahooweather_de.properties

```diff
@@ -1,3 +1,3 @@
-# binding
-binding.yahooweather.name = Yahoo Wetter Binding
-binding.yahooweather.description = Das Yahoo Wetter Binding stellt verschiedene Wetterdaten wie die Temperatur, die Luftfeuchtigkeit und den Luftdruck f�r konfigurierbare Orte vom yahoo Wetterdienst bereit
+# binding
+binding.yahooweather.name = Yahoo Wetter Binding
```

Comment:
We might want to remove "Yahoo" one day and replace these test sources by a "generic weather" binding.

Reply:
Yes or change it to a fictional ACME Weather Binding or something, which will keep providing weather services in the future. :-)